Extension/package.nls.json

```diff
@@ -162,6 +162,7 @@
     "c_cpp.configuration.vcpkg.enabled.markdownDescription": "Enable integration services for the [vcpkg dependency manager](https://aka.ms/vcpkg/).",
     "c_cpp.configuration.addNodeAddonIncludePaths.description": "Add include paths from nan and node-addon-api when they're dependencies.",
     "c_cpp.configuration.renameRequiresIdentifier.description": "If true, 'Rename Symbol' will require a valid C/C++ identifier.",
+    "c_cpp.configuration.autocomplete.addParenthesesAfterFunctionCalls.description": "If true, autocomplete will automatically add ( after function calls, in which case ) may also be added, depending on editor.autoClosingBrackets.",
```

Comment:
suggestion: "...depending on the value of the editor.autoClosingBrackets setting"

Reply:
Yeah, I was thinking the same thing. I wasn't sure which phrasing was better.